### ISSUE TITLE
Network Examples and lxcbird look like they are for just use w/LXC and not LXD?   Any chance of an LXD version?

### ISSUE THREAD

**Issue by bmullan** [knorrie/network-examples#8]
As you probably know, LXD and LXC are different.     Simos's blog post does a good explanation:

**[Comparison between LXC and LXD by Simos Xenitellis ](https://blog.simos.info/comparison-between-lxc-and-lxd/)**

Any chance of an "**_lxdbird_**"

thanks
brian

fyi I moderate both the [LXC sub-reddit](https://www.reddit.com/r/LXC/) and the [LXD sub-reddit](https://www.reddit.com/r/LXD/)
if you weren't aware of them..

**Comment by knorrie** [comment]
Hi,

Interesting. Can you point out what the actual advantages would be for this little tutorial project? The comparison blog tells me that you can do the exact same thing with slightly different looking commands.

When I can really spend some time again on this project, I badly want to start writing the next pages I have been having in mind for a long time already. I'm not really interested in redoing the container stuff in a different way if not needed.

I think 1) copy a template container to something else 2) start it 3) attach to it, mess around 4) stop it 5) destroy it is already pretty much the least amount of things you can end up with, which are also steps that are quite easy to understand. For just a little tutorial project, I guess we don't need any of the things that lxd claims to add on top.

Have fun,
Hans

P.S. the blog post uses 'lxc' for almost all the command examples in the lxd section.

**Comment by bmullan** [comment]
Yes, LXD's LXC use is (in my opinion) much more flexible & powerful than the Non-LXD LXC version.

Recent feature additions to LXD include new Network Interface types besides LXDBR0 (lxcbr0 in your case) that include 
routed interface (LXD container is connected to host default network by veth pair (one in host one in container)  
- macvlan
- OVN - open virtual network
- quite a few more

LXD supports the concept of "Projects" which I think is incredibly powerful and could allow you to create/add very unique new capabilities/features to your "network examples.   Reading through the following is pretty quick but provides a good overview:   
-  https://ubuntu.com/tutorials/introduction-to-lxd-projects#1-overview

Learning  LXD's LXC syntax is really quick if you are already familiar with the original LXC syntax.

Here is a project I spent time with over the past year using LXD, Wireguard, BGP, VXLAN, FRR (free range routing) to create an internet overlay system for LXD containers running on hosts that can be located pretty much anywhere:

https://github.com/bmullan/CIAB.Full-Mesh.VPN.Wireguard.FRR.BGP.VXLAN.Internet.Overlay.Architecture

If you look at the above note there are 2 youtube video's referenced.   The 2nd one has me installing/configuring everything in
the installation/configuration guide using 2 cloud LXD Server Hosts and a local VM hosting LXD.   I may have spent a long time at cisco but never much with BGP so if you do spot anything misdescribed/documented etc let me know :-)
